store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -284,6 +286,28 @@ public CarbonWriterBuilder withBlockSize(int blockSize) {
     return this;
   }
 
+  /**
+   * @param localDictionaryThreshold is localDictionaryThreshold,default is 1000
+   * @return
+   */
+  public CarbonWriterBuilder localDictionaryThreshold(int localDictionaryThreshold) {
+    if (localDictionaryThreshold <= 0) {
+      throw new IllegalArgumentException("blockSize should be between greater than 0");
```

Comment:
please change the error message accordingly. It is not a block size.

Reply:
Corrected .